CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Changelog
+
+## 0.1.1 (2021-09-02)
+
+- Update dependencies.
```

Comment:
Ditto. If users can use a USB camera by updating to 0.1.1, I think it is better to write that.